docs/contributing.rst

```diff
@@ -144,13 +145,13 @@ Running ``./test/isoltest`` again will result in a test failure:
             }
 
         Expected result:
-            DeclarationError: Identifier already declared.
+            DeclarationError: (36-52): Identifier already declared.
         Obtained result:
             Success
 
 
-which prints the expected result next to the obtained result, but also provides a way to change edit / update / skip the current contract or to even quit.
-``isoltest`` offers several options for failing tests:
+``isoltest`` prints the expected result next to the obtained result, but also provides a way to change edit / update / skip the current contract or to even quit.
+It offers several options for failing tests:
 
 - edit: ``isoltest`` will try to open the editor that was specified before using ``isoltest --editor /path/to/editor``. If no path was set, this will result in a runtime error. In case an editor was specified, this will open it such that the contract can be adjusted.
```

Comment:
We might want to add that - if the ``--editor`` option is not used -  ``isoltest`` will use the ``EDITOR`` environment variable, if set, and otherwise will try ``/usr/bin/editor``, if it exists (see https://github.com/ethereum/solidity/blob/b08d49338e2472ea579cbf8fcfb06a31bf517c1e/test/tools/isoltest.cpp#L283)).